README.md

```diff
@@ -46,82 +46,53 @@ If you wanted to install Chrome Beta and its associated driver:
+Node is required to run commands.
 
-- `curl` or `wget` to download files
-- `dpkg` or `rpm` to extract browser applications
-- `apt` or `yum` to install browsers
-- `unzip` to install ChromeDriver (if using the bash script directly)
+Install dependencies with
 
-Additionally, `sudo` permissions are needed in order to install browsers.
+`npm install`
 
 ## Commands and Options
 
 ### Commands
 
 - **install:** 
-    Install one or multiple browsers or drivers Can also pass the specific browser channel or driver version for each browser or driver. The latest Stable channel is used if no channel is passed. When installing ChromeDriver without passing a channel or version, the ChromeDriver version that matches the installed Stable version of Chrome is used.
+    Install the browser and driver. Can also pass the specific browser channel or version for each browser and driver. The latest Stable channel is used if no channel is passed.
 
     ```bash
-    # Install latest Chrome Stable
+    # Install latest Chrome Stable and matching Chromedriver version
     browser-driver-manager install chrome
 
-    # Install ChromeDriver version that matches install Chrome Stable
-    browser-driver-manager install chromedriver    
-
-    # Install latest Chrome Beta
-    browser-driver-manager install chrome=beta
-
-    # Install ChromeDriver version 97
-    browser-driver-manager install chromedriver=97
-
-    # Install both Chrome and ChromeDriver Beta
-    browser-driver-manager install chrome=beta chromedriver=beta
-    ```
+    # Install latest Chrome Beta and matching Chromedriver
+    browser-driver-manager install chrome@beta
 
 - **version:** 
-    Get the installed version of the browser or driver. Can also pass the specific browser channel.
+    Get the installed version of the browser or driver.
 
     ```bash
-    # Get installed Chrome version
-    browser-driver-manager version chrome
-
-    # Get installed Chrome Beta version
-    browser-driver-manager version chrome=beta
-
-    # Get ChromeDriver version
-    browser-driver-manager version chromedriver
-    ```
+    # Get installed Chrome and Chromedriver versions
+    browser-driver-manager version
 
 - **which:** 
-    Get the installed location of the browser or driver. Can also pass the specific browser channel.
+    Get the installed location of the browser and driver.
 
     ```bash
-    # Get installed Chrome location
-    browser-driver-manager which chrome
-
-    # Get installed Chrome Beta location
-    browser-driver-manager which chrome=beta
-
-    # Get ChromeDriver location
-    browser-driver-manager which chromedriver
+    # Get installed Chrome and Chromedriver locations
+    browser-driver-manager which
     ```
 
 ### Options
 
 - **-h,--help:** Display the help information
-- **-v,--version:** Display the version information
-- **--verbose:** Output verbose logs
```

Comment:
The code shows there's [a `verbose` option](https://github.com/straker/browser-driver-manager/pull/14/files#diff-7acf24c1eac65923388beac36a85ad14bc88a11ce203d01851cc6deea2c84df9R16)